[REF] Ported document_page_work_instructions

llacroix opened the pull request:
Depends on `document_page_approval`. Didn't require any other changes to work. 

Depends on 
- [x] OCA/knowledge#30
- [x] #20 

Edit add depenancies (yvaucher)

llacroix committed:
Ported document_page_work_instructions

llacroix committed:
Removed vim lines

dreispt commented:
We need `document_page_approval`ported ahead of this...

llacroix commented:
I'm aware of that. I used the branch that already ported `document_page_approval` to test the module. The PR is there when `document_page_approval` will be merged, it should work.

bwrsandman commented:
Depends on https://github.com/OCA/knowledge/pull/25

llacroix committed:
Moved description added licenses

llacroix committed:
Indent xml file to 2 spaces

llacroix committed:
Splitted xml file in views and data

dreispt commented on document_page_work_instructions/__openerp__.py:
```diff
@@ -27,10 +27,15 @@
+    ],
+    "data": [
+        'data/document_page.xml',
+
```
remove extra line

llacroix replied:
Is it necessary? I wanted to separate data data from view data.

dreispt replied:
I see. I would not add it, but I won't pick with that.

dreispt commented on document_page_work_instructions/data/document_page.xml:
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
```
noupdate="1"?

dreispt commented on document_page_work_instructions/views/document_page.xml:
```diff
@@ -0,0 +1,24 @@
+      <field name="view_type">form</field>
+      <field name="view_mode">tree,form</field>
+      <field name="view_id" ref="document_page.view_wiki_tree"/>
+      <field name="search_view_id" ref="document_page.view_wiki_filter"/>
```
I suggest removing these two *view_id attributes

max3903 commented on document_page_work_instructions/data/document_page.xml:
```diff
@@ -0,0 +1,26 @@
+
+<h1>Step 3</h1>
+        ]]>
+      </field>
```
This content can be moved in the category record above to serve as a template.

dreispt commented:
Minor notes, but globally :+1:

dreispt commented:
Needs one more review.

max3903 commented:
Waiting for the content to be moved in the category record and the content record to be removed.

llacroix committed:
Review update PR 26

Merge Category with Content Object of Document.page
and added noupdate=1 on data xml.

llacroix commented:
@max3903 , @dreispt  I updated the code according to your reviews. Anything else?

max3903 commented:
:+1:
